typegate/tests/runtimes/deno/deploy_deno.py

```diff
@@ -0,0 +1,40 @@
+from deno import deno
+
+import os
+import sys
+
+from typegraph.gen.exports.core import (
+    ArtifactResolutionConfig,
+    MigrationAction,
+    MigrationConfig,
+)
+from typegraph.graph.shared_types import BasicAuth
+from typegraph.graph.tg_deploy import TypegraphDeployParams, tg_deploy
+
+cwd = sys.argv[1]
+PORT = sys.argv[2]
+gate = f"http://localhost:{PORT}"
+auth = BasicAuth("admin", "password")
+
+deno_tg = deno()
+deploy_result = tg_deploy(
+    deno_tg,
+    TypegraphDeployParams(
+        base_url=gate,
+        auth=auth,
+        typegraph_path=os.path.join(cwd, "deno.py"),
```

Comment:
"deploy_deno.py"

Reply:
It's the path to the typegraph... `deploy_deno.py` is just the deploy script...